tests/parser/syntax/test_raw_call.py

```diff
@@ -23,17 +17,10 @@ def foo():
     (
         """
 @external
-def foo():
```

Comment:
why remove these two tests?

Reply:
These were probably in the wrong file. I will move them to `tests/parser/features/test_logging.py`.